portality/core.py

```diff
@@ -93,6 +93,12 @@ def configure_app(app):
         app.config.from_pyfile(config_path)
         print('Loaded secrets config from ' + config_path)
 
+    # import from instance.cfg
+    instance_path = os.path.join(proj_root, 'instance.cfg')
```

Comment:
please add `d) the ansible-generated instance config` to the docstring above

Reply:
added docstring